### PR TITLE
Add dropdown component

### PR DESCRIPTION
**Pull request by beckyvb**
Adds support for quick configuration of a sortable dropdown menu.

### CODE REVIEW

File: applications/dashboard/modules/class.dropdownmodule.php [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<?php if (!defined('APPLICATION')) exit();
```
Comment (linc) [inline applications/dashboard/modules/class.dropdownmodule.php]: We no longer add this to declaraction-only files.

File: applications/dashboard/modules/class.sortablemodule.php [hunk 2]
```diff
@@ -0,0 +1,467 @@
+<?php if (!defined('APPLICATION')) exit();
```
Comment (linc) [inline applications/dashboard/modules/class.sortablemodule.php]: same here

**Comment by linc** [comment]
This looks fine to me otherwise.

When adding this much code to core, I think it deserves more than a two-sentence description. ;) I was a bit put off from digging into this with no context or overview.

**Comment by linc** [comment]
`dropdown-twbs.php` - a typo of 'tabs' or an acronym I don't know?

**Comment by beckyvb** [comment]
Twitter Bootstrap. The view follows the CSS class naming conventions of Twitter Bootstrap. Thinking about it now, this doesn't need to be in core. An alternate dropdown view is nice to offer, but should live elsewhere.

**Pull request merged**